### PR TITLE
Hot Fix for issue77 headings

### PR DESCRIPTION
**Pull request by MikaelNordberg**

### CODE REVIEW

File: PXWeb/Selection.aspx [hunk 1]
```diff
@@ -11,14 +11,15 @@
     <meta property="og:site_name" content="<%= PXWeb.Settings.Current.General.Site.ApplicationName.ToString() %>" />
 </asp:Content>
 <asp:Content runat="server" ID="ContentTitle" ContentPlaceHolderID="TitlePlaceHolder">
-    <h2><asp:Label ID="MenuTitle" CssClass="hierarchical_tableinformation_title" runat="server" Text=""></asp:Label></h2>
+    <h2>
+        <asp:Label ID="MenuTitle" CssClass="hierarchical_tableinformation_title" runat="server" Text=""></asp:Label>
+        <pxc:TableInformation runat="server" Type="Normal" ID="TableInformationSelect" TableTitleCssClass="hierarchical_tableinformation_title" TableDescriptionCssClass="hierarchical_tableinformation_description"  EnableViewState="true" Visible="true" />
```
Comment (JohannesFinsveen) [inline PXWeb/Selection.aspx:16]: We should check the w3c validator to see that TableInformation only adds allowed elements in h2.

**Review by JohannesFinsveen — APPROVED** [review]
I have not run the code, just looked at it  :-)

**Review by PerIngeVaaje — COMMENTED** [review]
I have looked at the changes. Have not been involved in this part of code. Trust @JohannesFinsveen review  :)

**Review by likp — APPROVED** [review]
It sets the Heading as suggested.

**Pull request merged**